pkgs/by-name/ap/apt-mirror/package.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, perl
+, nix-update-script
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "apt-mirror";
+  version = "0.5.4";
+
+  src = fetchFromGitHub {
+    owner = "apt-mirror";
+    repo = "apt-mirror";
+    rev = finalAttrs.version;
+    hash = "sha256-GNsyXP/O56Y+8QhoSfMm+ig5lK/K3Cm085jxRt9ZRmI=";
+  };
+
+  nativeBuildInputs = [ perl ];
```

Comment:
the reason why autoPatchShebangs didn't work, because this needs to be `buildInputs`

here is copy from nixpkgs manual
https://nixos.org/manual/nixpkgs/stable/#ssec-stdenv-dependencies-overview
```
Add dependencies to nativeBuildInputs if they are executed during the build:
```

Reply:
That is it! Thanks again.